src/components/FeatureViews/Accounts/outOfOffice.js

```diff
@@ -0,0 +1,152 @@
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Dialog from '@material-ui/core/Dialog';
+import { DialogActions, DialogContent } from "@material-ui/core";
+import React, { useEffect, useMemo, useState } from "react";
+import Button from "@material-ui/core/Button";
+import { connect } from "react-redux";
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import { withStyles } from '@material-ui/core/styles';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import Grid from "@material-ui/core/Grid";
+import Checkbox from '@material-ui/core/Checkbox';
+import './Accounts.scss';
+
+const styles = {
+    minHeight: '80vh',
+    maxHeight: '80vh',
+};
+
+function OutOfOffice(props) {
+    console.log(props);
```

Comment:
Delete this. Also for future reference, if you need to view props, use React dev tools. 
<img width="1317" alt="Screen Shot 2020-02-25 at 3 51 10 PM" src="https://user-images.githubusercontent.com/12959959/75298286-ed16a400-57e6-11ea-9ca4-b20e3f744be2.png">